Source/HTTP/Curl/CurlMulti.cpp

```diff
@@ -9,6 +9,7 @@ namespace http_client
 
 // XCurl doesn't support curl_multi_timeout, so use a small, fixed delay between calls to curl_multi_perform
 #define PERFORM_DELAY_MS 50
+#define POLL_TIMEOUT_MS 10
```

Comment:
I think with the new model 0 is a reasonable choice.  I don't mind this, but it's now a 10ms sleep every time the work is queued.

Reply:
changed to 0